ubuntu-master/setup-bb-master-account.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
```

Comment:
Setting SCRIPT_DIR to not be the directory of the current script is confusing also.

I think what we really ought to have (here and elsewhere) is
ROOT_DIR=$(git rev-parse --show-toplevel), and we can then use includes relative to $ROOT_DIR everywhere.